focus-ios/Blockzilla.xcodeproj/project.pbxproj

```diff
@@ -6552,6 +6554,14 @@
 				version = 125.0.20240309050254;
 			};
 		};
+		8A0E7F2C2BA0F0E0006BC6B6 /* XCRemoteSwiftPackageReference "Fuzi" */ = {
+			isa = XCRemoteSwiftPackageReference;
+			repositoryURL = "https://github.com/nbhasin2/Fuzi.git";
+			requirement = {
+				branch = master;
+				kind = branch;
+			};
+		};
```

Comment:
Just double-checking: it looks like the Fuzi package was changed from exact version (3.1.3) to current master, was that intentional?

Reply:
Yes! To duplicate the Fuzi package dependency we have on Firefox iOS 👍